test/libsolidity/smtCheckerTests/imports/selected_contract.sol

```diff
@@ -0,0 +1,10 @@
+==== Source: A.sol ====
+contract A { function a() pure public { assert(false); } }
+==== Source: B.sol ====
+import "A.sol";
+contract B { function b() pure public { assert(false); } }
+// ====
+// SMTContract: B.sol:B
+// SMTEngine: chc
+// ----
+// Warning 6328: (B.sol:56-69): CHC: Assertion violation happens here.\nCounterexample:\n\n\nTransaction trace:\nB.constructor()\nB.b()
```

Comment:
IMO the purpose of this test would be clearer without the assert in `B.b()`. We'd then immediately see that no warning is generated instead of having to check where it's generated and whether that's expected or not.